src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import classNames from 'classnames';
+
+interface ErrorNotificationProps {
+  error: string;
+  onClose: () => void;
+}
+
+export const ErrorNotification: React.FC<ErrorNotificationProps> = ({
+  error,
+  onClose,
+}) => {
+  if (!error) return null;
+
+  return (
+    <div
+      className={classNames('notification', 'is-danger', 'is-light')}
```

Comment:
Do you need classNames here?
```suggestion
      className="notification is-danger is-light"
```